src/test/ui/suggestions/type-ascription-instead-of-statement-end.stderr

```diff
@@ -0,0 +1,16 @@
+error: expected type, found `0`
+  --> $DIR/type-ascription-instead-of-statement-end.rs:15:5
+   |
+14 |     println!("test"):
+   |                     - help: did you mean to end the statement here instead? `;`
```

Comment:
Huh, this looks strange to me somehow. Why not say "help: did you mean to use `;` here?"? I think users might not know that "end the statement" means `;` -- but also, I am not sure about this "floating" `;` that appears at the end of the sentence.